wukong/api.py

```diff
@@ -7,6 +7,14 @@
 
 logger = logging.getLogger(__name__)
 
+def _add_scheme_if_not_there(url, scheme='http'):
+    if not (
+        url.startswith('http://')
```

Comment:
I'm not sure if using something like `urlparse(url).scheme` would be better here, or if it would be overkill. I can't think of any particular case where somebody would use a scheme other than http or https, nor do I think solr supports it.

Reply:
I think that's fine